.travis.yml

```diff
@@ -3,16 +3,18 @@ sudo: required
 
 # Install Bazel and set up GOPATH.
 before_install:
-- go get -u github.com/alecthomas/gometalinter
-- wget https://github.com/bazelbuild/bazel/releases/download/0.4.4/bazel_0.4.4-linux-x86_64.deb
-- sudo dpkg -i bazel_0.4.4-linux-x86_64.deb
 - mkdir -p $GOPATH/src/k8s.io
 - mv $TRAVIS_BUILD_DIR $GOPATH/src/k8s.io
 - cd $GOPATH/src/k8s.io/repo-infra
+- echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
+- curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
+- sudo apt-get update
 
 install:
-  - gometalinter --install
+  - sudo apt-get install bazel
```

Comment:
It will break like it broke before when they remove debs. It's the worst of both worlds. We should consider rehosting on a gcs bucket we control.